solvcon/solver_legacy.py

```diff
@@ -42,6 +42,8 @@
 from .gendata import TypeWithBinder
 from .solver_core import ALMOST_ZERO
 
+import solvcon.mesh
```

Comment:
In solvcon package you should not use this full path importing style.  In this legacy file the style is to import the deepest symbol (which almost always isn't a module object).  In recent SOLVCON code I changed to import just the deepest module, and rename the imported module when necessary.

Please take a look at the context.